src/ontology/mondo-edit.obo

```diff
@@ -435508,6 +435508,17 @@ def: "A disease characteristic in which the disease has a known cause." [https:/
 is_a: MONDO:0700004 {source="https://github.com/monarch-initiative/mondo/issues/3177", source="https://orcid.org/0000-0002-4142-7153"} ! idiopathic vs non-idiopathic
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700007
+name: idiopathic disease
+def: "A disease or disorder for which the cause is of uncertain or unknown." [http://purl.obolibrary.org/obo/mondo/patterns/idiopathic.yaml, https://orcid.org/0000-0002-4142-7153]
```

Comment:
the dbxef should be:
MONDO:patterns/idiopathic, not http://purl.obolibrary.org/obo/mondo/patterns/idiopathic.yaml

Reply:
updated.